pkgs/development/libraries/libdrm/default.nix

```diff
@@ -1,18 +1,21 @@
-{ stdenv, fetchurl, pkgconfig, libpthreadstubs, libpciaccess, valgrind-light }:
+{ stdenv, fetchgit, meson, ninja, pkgconfig,
+  libpthreadstubs, libpciaccess, valgrind-light }:
 
 stdenv.mkDerivation rec {
-  name = "libdrm-2.4.97";
-
-  src = fetchurl {
-    url = "https://dri.freedesktop.org/libdrm/${name}.tar.bz2";
-    sha256 = "08yimlp6jir1rs5ajgdx74xa5qdzcqahpdzdk0rmkmhh7vdcrl3p";
+  name = "libdrm-${version}";
+  version = "2.4.97";
+  rev = "${name}";
+  sha256 = "065w0zyyfv3yzlw76pvsdjrkxd46wp5cwv472766is0mbi212rdp";
+
+  src = fetchgit {
+    url = "https://gitlab.freedesktop.org/mesa/drm";
+    inherit rev sha256;
```

Comment:
Why the change of source?

Reply:
It makes it easier to point at git revisions, but maybe not appropriate for this PR. I'll revert.